lib/config.ts

```diff
@@ -76,6 +79,7 @@ function getConfig(init: InitConfig): ResolvedConfig {
     experiments: init.experiments ?? DCN_DEFAULTS.experiments,
     mockedIP: init.mockedIP,
     sessionID: init.sessionID ?? generateSessionID(),
+    skipEnrichment: init.skipEnrichment ?? false,
```

Comment:
With suggestion above.
```suggestion
    skipEnrichment: init.skipEnrichment,
```